src/styles/Books.css

```diff
@@ -0,0 +1,191 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab:wght@700&display=swap");
+
+:root {
+  --black-two: #121212;
+  --cool-blue: #4386bf;
+  --grey-border: #e8e8e8;
+}
+
+.book {
+  flex-direction: column;
+  margin: 0 6.25rem 0.938rem 6.25rem;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px var(--grey-border);
+  background-color: var(--white);
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] I would suggest you adjust the spacing from top to button equally. If you see the logo, container, and Add-book then you can find this vertically in the same line.
- [ ] Kindly make the `BookStore CMS` bolder as the design.
### Figma design
![figma-design](https://user-images.githubusercontent.com/81727988/167230269-072e41b2-8659-4ead-9019-6bd3f36ca501.png)
### Your design
![his](https://user-images.githubusercontent.com/81727988/167230289-a4db3bb4-f95e-4e59-b5b4-65cc524dd3b4.png)